package.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "csbot",
+    "version": "1.0.0",
+    "description": "A bot to help manage the BYUCS Discord, successor to Ze Kaiser (https://github.com/arkenstorm/ze-kaiser)",
+    "main": "bot.js",
+    "scripts": {
+        "test": "echo \"Error: no test specified\" && exit 1"
+    },
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/BYU-CS-Discord/CSBot.git"
+    },
+    "author": "",
+    "license": "Unlicense",
+    "bugs": {
+        "url": "https://github.com/BYU-CS-Discord/CSBot/issues"
+    },
+    "homepage": "https://github.com/BYU-CS-Discord/CSBot#readme",
+    "devDependencies": {
+        "@types/sqlite3": "^3.1.8",
```

Comment:
Not sure, but figured I'd check, are these sqlite types necessary?

Reply:
They aren't for Prisma

Reply:
Oh there's a sqlite3 dependency below, is that necessary?

Reply:
Should I remove that line?

Reply:
If prisma takes care of the database connection I don't think sqlite is a necessary dependency, even if you use sqlite as the backend, idk for sure tho

Reply:
We'll get rid of it for now. Can always add it back later.

Reply:
Yeah I'm pretty sure the sqlite dependency isn't necessary to use sqlite with prisma